api.py

```diff
@@ -12,6 +12,20 @@
 def index():
     return 'Dino'
 
+
+@app.route('/submit-score', methods=['POST'])
+def submit_score():
+    data = request.json
+    username = data["username"]
+    score = data["score"]
+    userid = select("SELECT user_id FROM users WHERE username = %s", (username,))
+    format_user_id = userid[0][0]
```

Comment:
userid could be user_id (and on above line)

Reply:
👍